PaymentButtons/src/main/java/com/paypal/android/paymentbuttons/PayPalCreditButton.kt

```diff
@@ -62,33 +61,26 @@ class PayPalCreditButton @JvmOverloads constructor(
         )
         color = PayPalCreditButtonColor(attribute)
     }
+
+    companion object {
+        const val LUMINANCE_ERROR = "PayPalCreditButton does not have a light luminance compatible wordmark."
+    }
 }
 
 /**
  * Defines the colors available for PayPal Credit buttons.
  *
- * @see GOLD is the recommended color for PayPal Credit.
- * @see WHITE is a secondary alternative color for PayPal Credit.
- * @see DARK_BLUE is the default color if one is not provided. Deprecated - please use GOLD or WHITE.
- * @see BLACK is a secondary alternative color for PayPal Credit. Deprecated - please use GOLD or WHITE.
+ * @see DARK_BLUE is the default and recommended color for PayPal Credit.
+ * @see BLACK is a secondary alternative color for PayPal Credit.
  */
 enum class PayPalCreditButtonColor(
     val value: Int,
     override val colorResId: Int,
     override val hasOutline: Boolean = false,
     override val luminance: PaymentButtonColorLuminance
 ) : PaymentButtonColor {
-    @Deprecated("Deprecated color. Replace with gold or white.")
     DARK_BLUE(value = 0, colorResId = R.color.paypal_dark_blue, luminance = PaymentButtonColorLuminance.DARK),
-    @Deprecated("Deprecated color. Replace with gold or white.")
-    BLACK(value = 1, colorResId = R.color.paypal_black, luminance = PaymentButtonColorLuminance.DARK),
-    GOLD(value = 2, colorResId = R.color.paypal_gold, luminance = PaymentButtonColorLuminance.LIGHT),
```

Comment:
I don't think we can remove gold in a minor version if it's already been released without it being a breaking change

Reply:
Ah, true.
But how do we stop merchants from using it? It'll be an incompatible change and not be in parity with iOS.
I don't see a graceful way of handling this.

It is a breaking change, yes, but if we allow it, then it'll be an unexpected branding change. I guess breaking away from semver (unfortunately) is better than trying to talk our way out of branding issues.

I would love your thoughts on this.

Reply:
We can't stop merchants who are using `1.3.0` currently from using Gold and White, so the branding issue will exist either way. Can we deprecate Gold and White while un-deprecating the others? How are we handling it on iOS?

Reply:
iOS didn't release the changes, do they are in the clear to revert the changes and make it all go away.

Reply:
updated